sdk/core/azure-core/README.md

```diff
@@ -11,6 +11,8 @@ If you are a client library developer, please reference [client library develope
 | [Package (Conda)](https://anaconda.org/microsoft/azure-core/)
 | [API reference documentation](https://learn.microsoft.com/python/api/overview/azure/core-readme)
 
+This change will not be committed.
```

Comment:
`azure-core` has the additional dependencies, so we're touching this file (will not be merged) to ensure that having the linux-style paths will still resolve them in the common handlers.